0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -71,11 +71,14 @@ For more details on the flow, see the [Interledger Protocol specification](../00
 
 <span class="show alert alert-info">**Note:** Interledger only supports Universal mode as described in the whitepaper. Atomic mode can be used by adjacent subsets of participants in an Interledger payment if desired, but this is not part of the standard.</span>
 
-### Error Recovery and Connector Risk
+### Connector Risk and Mitigation
 
-An ILP execution chain can be interrupted if a connector fails to deliver the condition fulfillment. In this case, the sender thinks the transaction failed and retries it using the same condition as the original transaction. If the retry goes through the same connector that failed the first time, that connector can claim the incoming funds without sending a second outgoing transfer.
+Interledger protects senders from risk but connectors accept some risk in exchange for the revenue they generate from facilitating payments. In the Interledger payment flow, connectors' outgoing transfers are executed before their incoming transfers. Once each connector is notified that the outgoing transfer has been executed, they have a window of time to deliver the fulfillment and execute the incoming transfer. Connectors that fail to deliver the fulfillment in time may lose money.
```

Comment:
still, I think it's clear to say the refund guarantee is ultimately given by the ledger, and there are no mechanisms for claiming your refund if the ledger doesn't want to to give it to you. there is no way to 'let the network be the judge'. maybe under civil law you could claim your money back from the ledger, depending on their ToS? but then it's the law that gives the guarantee, not the Interledger network or the Interledger protocol.